clients/web/src/components/disconnect-button.tsx

```diff
@@ -9,13 +9,16 @@ export function DisconnectButton() {
     <Button.Root
       type="button"
       id="close"
-      class="border-gray-700 border rounded-md p-1 inline-flex items-center justify-center text-gray-400 hover:text-gray-500 hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-inset focus:ring-indigo-500"
+      class="border-slate-800 border w-6 hover:w-[8em] focus:w-[8em] transition-all duration-500 ease-in-out  rounded-md p-1 inline-flex items-center justify-end text-slate-200  focus:outline-none focus:ring-2 focus:ring-inset focus:ring-indigo-500 group relative overflow-x-hidden"
       onClick={() => {
         connectionStore.abortController.abort();
         goto("/");
       }}
     >
       <span class="sr-only">Close connection</span>
+      <span class="group-hover:inline-block group-focus:inline-block hidden absolute leading-[0] px-2 right-6 text-slate-200">
```

Comment:
Fixing the spacing would be nice:

![distance](https://github.com/crabnebula-dev/devtools/assets/119593300/bf919e62-8617-4548-9c3c-96cb8daaed65)

Reply:
good eye! Thanks!